Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -218,6 +231,10 @@ def mkOfLe {n} (i j : Fin (n+1)) (h : i ≤ j) : ([1] : SimplexCategory) ⟶ [n]
       | 0, 1, _ => h
   }
 
+/-- The morphism `[1] ⟶ [n]` that picks out the "diagonal composite" edge-/
+def mkOfDiag (n : ℕ) : ([1] : SimplexCategory) ⟶ [n] :=
```

Comment:
I am not sure `mkOfDiag` is the best name. What about `diag`?

Reply:
Done.

Reply:
Done.